djstripe/models/webhooks.py

```diff
@@ -295,7 +295,16 @@ def validate(
             # HTTP headers are case-insensitive, but we store them as a dict.
             headers = CaseInsensitiveMapping(self.headers)
             signature = headers.get("stripe-signature")
+            local_cli_signing_secret = headers.get(
+                "X-Djstripe-Local-Cli-Signing-Secret"
```

Comment:
make it all lowercase. It's case insensitive anyway but http2 headers are normalized to lowercase. Same for other references.

And let's rename it to `x-djstripe-webhook-secret`